streaming/README.md

```diff
@@ -316,6 +325,65 @@ With the introduction of TLS/SSL, the parent-child communication behaves as show
 | Yes|-/force/optional|Yes|no|The parent-child stream is encrypted, provided that the parent has a valid TLS/SSL certificate. Otherwise, the child refuses to connect.|
 | Yes|-/force/optional|Yes|yes|The parent-child stream is encrypted.|
 
+### Streaming compression
+Streaming compression is supported in Netdata streaming protocol version 5+.
+#### OS dependencies
+* Streaming compression is based on [lz4 v1.9.0+](https://github.com/lz4/lz4). The [lz4 v1.9.0+](https://github.com/lz4/lz4) library must be installed in your OS in order to enable streaming compression. Any lower version will disable Netdata streaming compression for compatibility purposes between the older versions of Netdata agents.
+
+You can check if your netdata agent supports and was build with stream compression through the Netdata agent web api (browser/terminal).
+```
+curl -X GET http://localhost:19999/api/v1/info | grep 'Stream Compression'
+```
+
+OUTPUT:
+<pre>
+"buildinfo": "dbengine|Native HTTPS|Netdata Cloud|ACLK Next Generation|New Cloud Protocol Support|ACLK Legacy|TLS Host Verification|Machine Learning|<b>Stream Compression</b>|protobuf|JSON-C|libcrypto|libm|LWS v3.2.2|mosquitto|zlib|apps|cgroup Network Tracking|EBPF|perf|slabinfo",
+</pre>
+Note: If your OS doesn't support Netdata compression the `buildinfo` will not contain the `Stream Compression` statement.
+
+You can check if your netdata agent has stream compression enabled or disabled.
+```
+ curl -X GET http://localhost:19999/api/v1/info | grep 'stream-compression'
+```
+OUTPUT:
+```
+"stream-compression": "enabled"
+```
+Note: The `stream-compression` status can be `"enabled" | "disabled" | "N/A"`.
+
+Stream data compression is enabled by default on systems where LZ4 library v1.9.0+ is installed. A compressed data packet is determined and decompressed on the fly.
+#### Limitations
+The current implementation of streaming data compression has the limitation that the size of single data block transmitted must not exceed 16384 bytes. If single data block size exceeds this limit, stream data compression should be disabled.
+#### How to enable stream compression.
+Netdata agents are shipped with data compression enabled by default but of course you can simply configure which streams will use compression.
+You can quickly enable compression in the `stream.conf` under the section `[stream]`.
+```
+# This is the default stream compression flag for an agent.
+
+[stream]
+enable compression = yes | no
+```
+Enabling compression will allow to the Netdata agent to negotiate streaming compression with other Netdata agents. During the negotiation of streaming compression BOTH Netdata agents should support and enable compression in order to communicate over a compressed stream. If one of the Netdata agents doesn't support OR has compression disabled then the negotiation will result in an uncompressed stream.
```

Comment:
@DShreve2 we are using uppercase to call attention here, is this the pattern we are using?

Reply:
You are right! No need to emphasize here with capital letters. I have included a detailed table that will do the same job.